### ISSUE TITLE
Is this extension unmaintained?

### ISSUE THREAD

**Issue by felipecrs** [mjbvz/vscode-lit-html#98]
The last update was in 2019.

I'm asking because now VS Code supports deprecating extensions: https://github.com/microsoft/vscode-discussions/discussions/1

There is another one, as a suggestion too: https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin

**Comment by mjbvz** [comment]
Yes if someone else can vouch for that extension I'd be happy deprecating this one. I don't use lit-html anymore in any of my projects

**Comment by felipecrs** [comment]
@runem, by any chance, do you know if lit-plugin supersedes lit-html?

**Comment by eric-burel** [comment]
I fill like runem.lit-plugin doesn't really format code. However it's good for syntax highlighting. So maybe adding formatting to this plugin could be relevant?

**Comment by vdegenne** [comment]
@felipecrs it doesn't, lit-plugin doesn't autocomplete custom element's attribute in html template which is unfortunate.

@mjbvz I'm also very curious what made you stop using this program of your own, as lit is still in active development at the moment.

**Comment by ivanjeremic** [comment]
This extension is what you guys should switch to [https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin) It just works and is awesome!

**Comment by metaleap** [comment]
This [fork of this repo](https://github.com/ericc-ch/vscode-lit-html) works excellently to this day, btw .. great for people not using Lit but still using `html´..´` lits (like me). Big thanks to both @mjbvz and @ericc-ch !
